mps_youtube/config.py

```diff
@@ -302,6 +314,10 @@ class _Config:
             ConfigItem("window_size", "",
                 check_fn=check_win_size, require_known_player=True),
             ConfigItem("download_command", ''),
+            ConfigItem("lastfm_username", ''),
+            ConfigItem("lastfm_password", '', check_fn=check_lastfm_password),
+            ConfigItem("lastfm_api_key", ''),
+            ConfigItem("lastfm_api_secret", ''),
```

Comment:
Adding not only one but four configuration options just for this isn't great, although I don't know if there's a better way to do it. I guess it might be better if mpsyt had multiple sections for settings or something like that...

This could by an issue if the list output by `set` doesn't fit on a page; it isn't currently paginated. Of course, that issue isn't specific to this change.

Reply:
Oh yes that reminds me, I found 4 options to be a lot as well. Unfortunately, I don't seen any distinctly better solutions either. Maybe the information could be read from a config file ? I think [cmusfm](https://github.com/Arkq/cmusfm) does it that way. It could for example read the last.fm information from a file in `$HOME/.config/mps-youtube/`. 

There is also some ENV variables pylast looks for/sometimes uses in example code, so maybe we could use those (you would have to do something like `export PYLAST_USERNAME=foo`, for example). On the other hand, seemingly nobody else seems to use those variables.